contracts/mixnet/src/contract.rs

```diff
@@ -201,11 +201,13 @@ pub fn query(deps: Deps, _env: Env, msg: QueryMsg) -> Result<QueryResponse, Cont
             gateway_identity,
             address,
         )?),
+        QueryMsg::GetTotalMixStake {} => to_binary(&queries::query_total_mix_stake(deps)),
     };
 
     Ok(query_res?)
 }
 
+// TODO: Calculate initial total stake when deploying this change
```

Comment:
I'd perhaps move it explicitly to migrate via `todo!()` macro, so that migration would fail to run without this change (in case somebody misses the comment and wanted to run it)

Reply:
I had no idea `todo!()` was a thing :)